src/module/canvas/layer/template.ts

```diff
@@ -16,7 +16,7 @@ export class TemplateLayerPF2e<
     /** Overriden to snap according to the dragged template's type */
     override getSnappedPoint(point: Point): Point {
         const template = this.preview.children.at(0);
-        if (!template || !canvas.grid.isSquare) {
+        if (!template || canvas.grid.isGridless) {
```

Comment:
We'll want to keep this so that hex snapping is retrieved from upstream.

Reply:
The reason for using snappingMode on hex grids is that other parts of the code already do so, so avoiding it's use in this one case can lead to the wrong assumption about it's use. I can alternately go to each use of snappingMode and add if checks so it is only called on square grids.

Reply:
`_getGridHighlightPositions` currently uses snappingMode on hex
`_onDragLeftMove` currently uses snappingMode on hex
`activatePreviewListeners` currently uses snappingMode on hex

Reply:
Those will all need to follow suit.

Reply:
Okay I'll go ahead and make sure that hex does not use `snappingMode`

Reply:
Okay I changed the two locations that use snappingMode when not on square... Let me know if you want me to format it differently.